src/main/java/seedu/address/logic/commands/LockCommand.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PASSWORD;
+
+import java.nio.charset.StandardCharsets;
+import java.util.logging.Logger;
+
+import com.google.common.hash.Hashing;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.UserPrefs;
+
+//@@author charlesgoh
+/**
+ * Sets lock in model to true or false, which allows or restricts the usage of all commands
+ */
+public class LockCommand extends Command {
+
+    public static final String COMMAND_WORD = "lock";
+    public static final String COMMAND_ALIAS = "lk";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Locks the application. "
+            + "No commands can be executed\n"
+            + "Parameters: "
+            + "[" + PREFIX_PASSWORD + "PASSWORD] ";
```

Comment:
[ ] around password.

Reply:
I thought [] meant optional? Default password is "password" so users will have to type in everything right?